src/Essentials/src/SecureStorage/SecureStorage.ios.tvos.watchos.macos.cs

```diff
@@ -9,8 +9,24 @@ namespace Microsoft.Maui.Essentials
 {
 	public static partial class SecureStorage
 	{
-		public static SecAccessible DefaultAccessible { get; set; } =
-			SecAccessible.AfterFirstUnlock;
+		public static SecAccessible DefaultAccessible
+		{
+			get { return Implementations.SecureStorageImplementation.DefaultAccessible; }
+			set { Implementations.SecureStorageImplementation.DefaultAccessible = value; }
+		}
+
+		public static Task SetAsync(string key, string value, SecAccessible accessible) =>
+			Implementations.SecureStorageImplementation.SetAsync(key, value, accessible);
```

Comment:
Maybe put this on `ISecAccessibleSecureStorage : ISecureStorage`

Reply:
Maybe if `Current is not ISecAccessibleSecureStorage` then we can throw a `NotImplementedException`.

Reply:
Addressed in 0c43ef1. Note that `ISecAccessibleSecureStorage` has to be a platform-dependent type, otherwise we would have to mirror the `SecAccessible` enum as a "cross-platform" type.